### PR TITLE
adressRessourceOk

### PR DESCRIPTION
**Pull request by JIdayyy**

### CODE REVIEW

File: src/api/address/controller.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import UserHandlers from "./interfaces";
```
Comment (Karnak19) [inline src/api/address/controller.ts]: Rename this

File: src/api/address/delete.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import prisma from "../../../prisma/prismaClient";
+
+import PropertyHandlers from "./interfaces";
```
Comment (Karnak19) [inline src/api/address/delete.ts:3]: Rename also this

File: src/api/address/interfaces.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { Address, Property } from ".prisma/client";
+import { RequestHandler } from "express";
+
+interface ReqBodyAddressPost {
+  id :          string;
```
Comment (Karnak19) [inline src/api/address/interfaces.ts]: No ID in this Interface

File: src/api/index.ts [hunk 4]
```diff
@@ -2,10 +2,12 @@ import { Router } from "express";
 
 import users from "./users/routes";
 import properties from "./properties/routes";
+import address from "./address/routes"
 
 const router = Router();
 
 router.use("/users", users);
 router.use("/properties", properties);
+router.use("/address", address);
```
Comment (Karnak19) [inline src/api/index.ts]: addresses ? Plural ;)

**Review by Karnak19 — CHANGES_REQUESTED** [review]

**Comment by JIdayyy** [comment]
Fix ok !

**Pull request merged**